ci/test/scaffold

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -ex
+
+source ./common
+init_test_repo
+cd $test_repo_path
+
+function check_file(){
+    if [ -e $1 ]
+then
+    echo "Success ! $1 was present"
+else
+    exit 1
+fi
+
+}
+
+popper init
+
+popper scaffold
+check_file $test_repo_path/main.workflow
```

Comment:
given that we are doing `set -ex` at the beginning of the script, this can be done by doing:

```bash
test -f $test_repo_path/main.workflow
```